vpn/tunnel.go

```diff
@@ -111,8 +148,12 @@ func (t *Tunnel) handleRPC(req *ManagerMessage, msgID uint64) *TunnelMessage {
 		if err != nil {
 			t.logger.Critical(t.ctx, "failed to get current workspace state", slog.Error(err))
 		}
+		update, err := t.createPeerUpdate(state)
+		if err != nil {
+			t.logger.Error(t.ctx, "failed to populate agent network info", slog.Error(err))
+		}
 		resp.Msg = &TunnelMessage_PeerUpdate{
-			PeerUpdate: convertWorkspaceUpdate(state),
+			PeerUpdate: update,
```

Comment:
Is it OK to send a nil `PeerUpdate` in the response?

Reply:
Good catch, I don't think we should. If there's no active connection (for whatever reason, should be impossible once the tunnel has started), we'll just send a nil last handshake time.